### PR TITLE
Avoid deeply nesting log file when running tests

### LINKED ISSUE #40196: quarkus.log.file.path set to a deeply nested `target/.../target/quarkus.log`
I had a look at a Surefire report on CI and the `quarkus.log.file.path` was set to the following value as reported by Surefire in the test report:

```xml
<property name="quarkus.log.file.path" value="target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/target/quarkus.log"/>
```

This is very odd.

Note that the number of nested targets depend on the test. Some have more of them and some less. I wonder if we are adding a `target` for each test run.

### PR DESCRIPTION
**Pull request by gsmet**
Fixes #40196

### CODE REVIEW

File: test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java [hunk 1]
```diff
@@ -22,23 +22,17 @@ public static void setLogFileProperty(String logFileName) {
...
-        List<String> logFileLocationParts = getLogFileLocationParts(getLogFinalName());
-        if (logFileLocationParts.isEmpty()) {
-            throw new IllegalStateException("Unable to determine log file path");
-        } else if (logFileLocationParts.size() == 1) {
-            return Paths.get(logFileLocationParts.get(0));
-        } else {
-            return Paths.get(logFileLocationParts.get(0),
-                    logFileLocationParts.subList(1, logFileLocationParts.size()).toArray(new String[0]));
-        }
+        return Paths.get(getLogFileLocation());
```
Comment (gsmet) [inline test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java:35]: Interesting in your feedback on this particular change as I didn't understand the need for such complexity. But maybe you tried to work around the issue I fixed above.

Comment (geoand) [reply test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java:35]: I honestly don't remember, but it certainly could be the case.

If CI is all good, then I have no objections :)

**Review by geoand — APPROVED** [review]

File: test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java (hunk 1 above)
Comment (gsmet) [reply test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java:35]: That, we will discover soon :).

Comment (geoand) [reply test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java:35]: 🤞🏼

**Pull request merged**